client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -244,68 +263,319 @@ class EmailProvidersComparison extends React.Component {
+				onExpandedChange={ this.onExpandedStateChange }
+				formattedPrice={ formattedPrice }
+				discount={ discount }
+				formFields={ formFields }
+				buttonLabel={ translate( 'Add Email' ) }
+				onButtonClick={ this.onAddTitanClick }
+				features={ this.getTitanFeatures() }
+			/>
+		);
+	}
+
+	onGoogleUsersChange = ( changedUsers ) => {
+		this.setState( { googleUsers: changedUsers } );
+	};
+
+	onGoogleFormReturnKeyPress = ( event ) => {
+		// Simulate an implicit submission user form :)
+		if ( event.key === 'Enter' ) {
+			this.onGoogleConfirmUsers();
+		}
+	};
+
+	onGoogleConfirmNewUsers = () => {
+		const { googleUsers } = this.state;
+
+		const usersAreValid = areAllUsersValid( googleUsers );
+		if ( ! usersAreValid ) {
+			return;
+		}
+
+		const { domain, productsList, selectedSiteSlug, shoppingCartManager } = this.props;
+		const domains = [ domain ];
+		const googleProductSlug = config.isEnabled( 'google-workspace-migration' )
+			? GOOGLE_WORKSPACE_BUSINESS_STARTER_YEARLY
+			: GSUITE_BASIC_SLUG;
+
+		shoppingCartManager
+			.addProductsToCart(
+				getItemsForCart( domains, googleProductSlug, googleUsers ).map( ( item ) =>
+					fillInSingleCartItemAttributes( item, productsList )
+				)
+			)
+			.then( () => {
+				this.isMounted && page( '/checkout/' + selectedSiteSlug );
+			} );
+	};
+
+	renderStackedGoogleDetails() {
+		const { domain, isGSuiteSupported } = this.props;
+
+		if ( ! isGSuiteSupported ) {
+			return null;
+		}
+
+		const googleProps = this.getGoogleDetailProps();
+		// If we don't have any users, initialize the list to have 1 empty user
+		const googleUsers =
+			( this.state.googleUsers ?? [] ).length === 0
+				? newUsers( domain.name )
+				: this.state.googleUsers;
+
+		const formFields = (
+			<FormFieldset>
+				<GSuiteNewUserList
+					extraValidation={ noop }
+					domains={ [ domain ] }
+					onUsersChange={ this.onGoogleUsersChange }
+					selectedDomainName={ domain.name }
+					users={ googleUsers }
+					onReturnKeyPress={ this.onGoogleFormReturnKeyPress }
+				/>
+			</FormFieldset>
+		);
+
+		return (
+			<EmailProviderCard
+				providerKey="google"
+				logo={ { path: googleProps.productLogo } }
+				formFields={ formFields }
+				detailsExpanded={ this.state.expanded.google }
+				onExpandedChange={ this.onExpandedStateChange }
+				onButtonClick={ this.onGoogleConfirmNewUsers }
+				buttonDisabled={ ! areAllUsersValid( googleUsers ) }
+				features={ this.getGoogleFeatures() }
+				{ ...googleProps }
 			/>
 		);
 	}
 
+	renderStackedForwardingDetails() {
+		const { domain, translate } = this.props;
+
+		const formFields = (
+			<>
+				<FormFieldset>
+					<FormLabel>
+						{ translate( 'Emails sent to' ) }
+						<FormTextInputWithAffixes
+							required
+							suffix={ `@${ domain.name }` }
+							onChange={ this.onForwardingEmailChange }
+						/>
+					</FormLabel>
+				</FormFieldset>
+				<FormFieldset>
+					<FormLabel>
+						{ translate( 'Will be forwarded to' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( 'Will be forwarded to', { context: 'Form label: (Email sent to [address]) will arrive at [address]'} )` *ES Score: 15*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/updated-email-comparison-page)